packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsResetPasswordDialog.tsx

```diff
@@ -45,23 +46,23 @@ const renderActions = (
 export const DatabaseSettingsResetPasswordDialog: React.FC<Props> = (props) => {
   const { open, onClose, databaseEngine, databaseID } = props;
 
-  const [isLoading, setIsLoading] = React.useState(false);
-  const [error, setError] = React.useState('');
+  const [error, setError] = React.useState<string | undefined>();
+
+  const { mutateAsync, isLoading } = useDatabaseCredentialsMutation(
+    databaseEngine,
+    databaseID
+  );
```

Comment:
```suggestion
  const { mutateAsync, isLoading, error } = useDatabaseCredentialsMutation(
    databaseEngine,
    databaseID
  );
```

Source the error from here and remove the `error` state.